Physical_Engine/Query/VolumetricMaterialTakeoff.cs

```diff
@@ -149,6 +149,45 @@ public static VolumetricMaterialTakeoff VolumetricMaterialTakeoff(this Door open
             return TakeOff(opening.Location.IArea(), opening.Construction);
         }
 
+        /***************************************************/
+
+        [Description("Gets the volumetric material takeoff from a PadFoundation.")]
+        [Input("padFoundation", "The physical PadFoundation object to extract the volumetric material takeoff from.")]
+        [Output("volTakeoff", "The volumetric material takeoff for the PadFoundation, made of up the volume and materiality of the construction and surface area.")]
+        public static VolumetricMaterialTakeoff VolumetricMaterialTakeoff(this PadFoundation padFoundation)
+        {
+            if (padFoundation.IsNull())
+                return null;
+
+            return TakeOff(padFoundation.Location.IArea(), padFoundation.Construction);
```

Comment:
```suggestion
            return TakeOff(padFoundation.Location.IArea(), padFoundation.Construction);
```

Reply:
Was this comment complete? Looks the same to me.

Reply:
I do not recall attempting to make any comment on these lines - looks like a missclick or some other glitch on my end to me 🤦‍♂️ Apologies